test/groovy/CloudFoundryDeployTest.groovy

```diff
@@ -86,13 +86,12 @@ class CloudFoundryDeployTest extends BasePiperTest {
             deployTool: '',
             stageName: 'acceptance',
         ])
-
-        assertTrue(jlr.log.contains('[cloudFoundryDeploy] General parameters: deployTool=, deployType=standard, cfApiEndpoint=https://api.cf.eu10.hana.ondemand.com, cfOrg=testOrg, cfSpace=testSpace, cfCredentialsId=myCreds, deployUser=testUser'))
```

Comment:
unrelated to scope of PR, only another way of asserting --> should be done in a dedicate PR.